File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    var answer = '';
    let skipArr = skip.split('');
    
    for(let i=0; i<s.length; i++){
        let ch = s[i];
        let moved = 0;

        while(moved<index){
            ch = String.fromCharCode(ch.charCodeAt(0)+1);
            
            if(ch >'z') ch = 'a';
            
            if(skipArr.includes(ch)){
                continue;
            }

            moved++;
        }
        answer+=ch
    }
    
    return answer;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    for ch in s:
        moved = 0
        while moved < index:
            ch = chr(ord(ch)+1)
            
            if ch > 'z': ch = 'a'
            
            if ch in skip:
                continue
            
            moved+=1
        answer+=ch        
        
    return answer